src/hooks/useTransactionSteps.ts

```diff
@@ -121,42 +117,43 @@ const useTransactionSteps = ({ transaction, onComplete }: Props) => {
       {
         chainType: targetChain,
         label: 'Deliver message',
-        labelOnChain: onChainCompleted(messageDelivered) && transaction.messageNonce,
+        labelOnChain: onChainCompleted(messageDelivered) && deliveryBlock,
         status: completionStatus(messageDelivered)
       },
       {
         chainType: targetChain,
         label: 'Finalize message in target block',
-        labelOnChain: onChainCompleted(messageFinalizedOnTarget) && targetMessageDeliveryBlock,
+        // TODO: To apply a different design for blocks and nonces: #175
+        labelOnChain: onChainCompleted(messageFinalizedOnTarget) && deliveryBlock,
```

Comment:
We shouldn't be displaying the same block twice (first in delivery then in finality), let's just not display anything as in source finality.

Reply:
why duplicating? in the first step we show the finalized block on the source and in this step we show the finality block of the target...

If you feel showing the information of finalized blocks on both  chains is redundant , then i can remove it with no issues. @goldsteinsveta , do you have any comments on this?

Reply:
> why duplicating? in the first step we show the finalized block on the source and in this step we show the finality block of the
 
![image](https://user-images.githubusercontent.com/581548/118715728-40c64080-b824-11eb-949e-26b9459e53fa.png)

Do you see `6405` twice? That's exactly the duplication I mean.

We don't display `6161` twice - it's only included when message is included in block.

Perhaps we should change labels to:
```
- Deliver message in target block 6405
- Finalize message
```
to make it even more clear.

Displaying `nonce` is only useful for debugging really - it should be of no use to the end user, I'd suggest putting it to a tooltip (or some future place where we display more details about the transaction).

Reply:
ouh! i didn't catch that ... actually it was a regression on this PR. 

The main idea is:
- Deliver message  <NONCE>
- Finalize message in target block <BLOCK>

as soon as i see we are not showing the message nonce anywhere.

Reply:
never mind ... your proposal makes sense.